src/ildasm/dasm.cpp

```diff
@@ -36,12 +36,12 @@
 #include <corcompile.h>
 #endif
 
+#if !FEATURE_CORECLR
```

Comment:
The common style for ifdefs in the CoreCLR codebase is `#ifdef FEATURE_CORECLR / #ifndef FEATURE_CORECLR`